packages/types/src/verification.ts

```diff
@@ -62,4 +62,11 @@ export enum FieldError {
   InvalidBankName = "invalid-bank-name",
   ShortTopicName = "short-topic-name",
   LongTopicName = "long-topic-name",
+  ShortContactRequestTitle = "short-contact-request-title",
+  LongContactRequestTitle = "long-contact-request-title",
+  ShortContactRequestMessage = "short-contact-request-message",
+  LongContactRequestMessage = "long-contact-request-message",
+  InvalidName = "invalid-name",
+  ShortName = "short-name",
+  LongName = "long-name",
```

Comment:
Any reason why not using these https://github.com/litespace-org/litespace/blob/fa0c6077914cadfa43b0d5d000650d50da5ccf85/packages/types/src/verification.ts#L6-L8